documentation/user_docs/docs/manual/Binary_operations.rst

```diff
@@ -2,15 +2,18 @@
 Binary operations
 #################
 
+Introduction
+============
+
 Binary operations between two objects can be applied in a variety of ways in
 Horace. You can either use the symbolic form (``+``, ``-`` , ``*``, ``/``,
 ``\``), or you can use the explicit function names (``plus, minus, mtimes,
 mrdivide``, ``mldivide``, etc.).
 
-Main Horace data objects (main from binary operation point of view) have binary operations defined between them
+Main from binary operation point of view Horace data objects have binary operations defined between them
```

Comment:
I preferred the original text. There are languages where this structure works, but not in English